### PR TITLE
hw-01 submission Hengle Li

### PR DESCRIPTION
**Pull request by atani21**
Easy enough for a first homework of the quarter. I am wondering, though, since we need a link to insert any image. Does that mean if we want to insert any image in these files, the image has to be stored somewhere online beforehand?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+### **basics**
+* Name: Hengle Li (Eric)
+* Year: first-year MAPSS
+* Concentration: Sociology
+
+#### _a few words_
+Hello, my name is Hengle Li. I also go by Eric, because there are so many various pronunciations of my first name, I don't want to bother anymore. 
+I am from China, and I went to college in UC Berkeley, as a sociology major. My research interests are social change, sociological theories, and political sociology. Bourdieu is my favorite, though I hate his writing style. After Bourdieu, I like Marx the best, because he's one of the few sociologists that know how to write properly for people to read their texts.  
+I have no programming experience before, so all these are like dark magic to me. Cheers!
+
```
Comment (russell-roberts) [inline README.md:12]: Programming is like dark magic to me too! Hopefully this course will help us become practitioners.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+![my photo](https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/22886269_1981378462118343_6779231979075832945_n.jpg?_nc_cat=107&_nc_oc=AQmiGwbjkz7ysKtsfUufMHsYhumTkmFJLz88NomnObfl46qNJsOmP4yuif76TDN3-WE&_nc_ht=scontent-ort2-2.xx&oh=a69a4d144d36e97f13c224e3c216040e&oe=5E37D896)
+
```
Comment (russell-roberts) [inline README.md:14]: Nice photo! Looks like those giraffes live in a house that is nicer than my apartment.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+# hw01
+## biography
+### **basics**
+* Name: Hengle Li (Eric)
+* Year: first-year MAPSS
+* Concentration: Sociology
+
+#### _a few words_
+Hello, my name is Hengle Li. I also go by Eric, because there are so many various pronunciations of my first name, I don't want to bother anymore. 
+I am from China, and I went to college in UC Berkeley, as a sociology major. My research interests are social change, sociological theories, and political sociology. Bourdieu is my favorite, though I hate his writing style. After Bourdieu, I like Marx the best, because he's one of the few sociologists that know how to write properly for people to read their texts.
```
Comment (russell-roberts) [inline README.md:10]: I haven't read any sociological theory for a long time, but I remember liking Marx too. What are your thoughts on Durkheim?

**Review by russell-roberts — COMMENTED** [review]
Hi Eric, 

Looks like you've done a good job incorporating text features into R Markdown.  I'm particularly impressed with your embedded photo--those giraffes are really living the good life!

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+* Name: Hengle Li (Eric)
+* Year: first-year MAPSS
+* Concentration: Sociology
+
+#### _a few words_
+Hello, my name is Hengle Li. I also go by Eric, because there are so many various pronunciations of my first name, I don't want to bother anymore. 
+I am from China, and I went to college in UC Berkeley, as a sociology major. My research interests are social change, sociological theories, and political sociology. Bourdieu is my favorite, though I hate his writing style. After Bourdieu, I like Marx the best, because he's one of the few sociologists that know how to write properly for people to read their texts.  
+I have no programming experience before, so all these are like dark magic to me. Cheers!
+
+![my photo](https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/22886269_1981378462118343_6779231979075832945_n.jpg?_nc_cat=107&_nc_oc=AQmiGwbjkz7ysKtsfUufMHsYhumTkmFJLz88NomnObfl46qNJsOmP4yuif76TDN3-WE&_nc_ht=scontent-ort2-2.xx&oh=a69a4d144d36e97f13c224e3c216040e&oe=5E37D896)
```
Comment (zxgneil) [inline README.md:13]: Try using a relative path & upload your picture here. That will help simplify your code a bit.

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |         x     |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |           |       x       |            |

Remarks:
1. You experimented with more than four ways of using markdown. Good work!
1. I wish there is more commit history for easier access. 
1. Using a relative file path will help simplify your code a bit more.

**Pull request closed without merging**